.github/workflows/deploy.yml

```
@@ -0,0 +1,148 @@
+name: Vercel Deploy (NeonPro)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)